### PR TITLE
feat(hapi): send rejection mail

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Notify the user if they are rejected by KYC requirement

### Steps to test

1. `make run`

### CODE REVIEW

File: hapi/src/utils/templates/mail.template.js [hunk 1]
```diff
@@ -74,6 +74,79 @@ const generateConfirmationMail = ({ account }) => {
...
+                      <a href="https://earnproton.com" style="text-decoration: none; margin: 40px 0px 0px 16px; width: 178px; height: 53px;">
+                          <img style="object-fit: scale-down;" src="https://earnproton.com/icons/proton.png"/>
+                      </a>
+                      <p align="left" style="margin: 24px 16px 24px; font-family: Arial; font-size: 21px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: normal; letter-spacing: 0.15px; text-align: left; color: #000;">
+                          ${account} - Some News Regarding Your Proton On-Chain Referral Program Request
+                      </p>
+                      <p align="left" style="margin: 0px 16px 8px; font-family: Arial; font-size: 16px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: 1.5; letter-spacing: 0.44px; text-align: left; color: #000;">
+                        Your Proton referral link has not been approved. To begin using the Proton Referral Program and start earning rewards, you must first complete the KYC (Know Your Client) authentication process in the Proton wallet.
+                      </p>
```
Comment (xavier506) [inline hapi/src/utils/templates/mail.template.js:96]: should we mention which countries have KYC available?


There is going to be an update soon to the proton wallet which enables many more countries for KYC , should we tell users about this?

File: hapi/src/utils/templates/mail.template.js [hunk 2]
```diff
@@ -74,6 +74,79 @@ const generateConfirmationMail = ({ account }) => {
...
+                      <br>
+                      <p align="left" style="margin: 0px 16px 24px; font-family: Arial; font-size: 16px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: 1.5; letter-spacing: 0.44px; text-align: left; color: #000; overflow-wrap: break-word;">
+                        To request joining the Referral Program, go to the Proton wallet app and submit your KYC application. If you're still having the same problem, please get in touch with us through Telegram: https://t.me/eoscr.
```
Comment (xavier506) [inline hapi/src/utils/templates/mail.template.js:99]: If users are having trouble with KYC they should not be sent to our telegram as we dont handle KYC process

maybe we should send them to the general XPR chat

File: hapi/src/utils/templates/mail.template.js [hunk 3]
```diff
@@ -74,6 +74,79 @@ const generateConfirmationMail = ({ account }) => {
...
+      </head>
+      <body>
+          <table style="margin-left: auto; margin-right: auto;">
+              <tr>
+                  <div style="max-width: 640px; margin-left: auto; margin-right: auto;">
+                      <a href="https://earnproton.com" style="text-decoration: none; margin: 40px 0px 0px 16px; width: 178px; height: 53px;">
+                          <img style="object-fit: scale-down;" src="https://earnproton.com/icons/proton.png"/>
+                      </a>
+                      <p align="left" style="margin: 24px 16px 24px; font-family: Arial; font-size: 21px; font-weight: normal; font-stretch: normal; font-style: normal; line-height: normal; letter-spacing: 0.15px; text-align: left; color: #000;">
+                          ${account} - Some News Regarding Your Proton On-Chain Referral Program Request
```
Comment (xavier506) [inline hapi/src/utils/templates/mail.template.js:92]: I would eliminate "Some News"

Regarding Your Proton On-Chain Referral Program Request

File: webapp/src/routes/Admin/index.js [hunk 4]
```diff
@@ -343,6 +347,13 @@ const Admin = () => {
 
   const deleteNewUsers = async (showSnack = true) => {
     try {
+      // DON'T PLACE THIS CALL AFTER DELETE ACTION TO ENSURE THE USERS'S MAIL STILL EXIST
```
Comment (xavier506) [inline webapp/src/routes/Admin/index.js:350]: do we need this comment?

**Review by xavier506 — CHANGES_REQUESTED** [review]
view comments

File: hapi/src/utils/templates/mail.template.js (hunk 2 above)
Comment (leisterfrancisco) [reply hapi/src/utils/templates/mail.template.js:99]: do you mean [this group](https://t.me/protonxpr)

File: hapi/src/utils/templates/mail.template.js (hunk 3 above)
Comment (leisterfrancisco) [reply hapi/src/utils/templates/mail.template.js:92]: `${account} -` is a pattern used in the mail template, would you want me to update this pattern?

Comment (leisterfrancisco) [reply hapi/src/utils/templates/mail.template.js:92]: Current template is:

`${account} -` ...........

File: hapi/src/utils/templates/mail.template.js (hunk 1 above)
Comment (leisterfrancisco) [reply hapi/src/utils/templates/mail.template.js:96]: New content

File: hapi/src/utils/templates/mail.template.js (hunk 3 above)
Comment (leisterfrancisco) [reply hapi/src/utils/templates/mail.template.js]: @xavier506 ?

**Comment by leisterfrancisco** [comment]
image

@xavier506 this is how it looks now.

**Comment by leisterfrancisco** [comment]
These PR changes will be included in a new PR.

**Pull request closed without merging**